### PR TITLE
Feature/open api/swagger ui

### PR DESCRIPTION
**Pull request by inngvar**
https://github.com/jhipster/jhipster-quarkus/issues/6
Simple integration with minimal changes

### CODE REVIEW

File: generators/server/templates/quarkus/src/main/java/package/service/dto/ManagementInfoDTO.java.ejs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import javax.json.bind.annotation.JsonbProperty;
+import java.util.List;
+
+/**
+*  DTO to emulate /management/info response
+*/
+public class ManagementInfoDTO {
+
+    public List<String> activeProfiles = Lists.newArrayList();
```
Comment (danielpetisme) [inline generators/server/templates/quarkus/src/main/java/package/service/dto/ManagementInfoDTO.java.ejs:32]: What are the benefits of Google `Lists.newArrayList()` vs the JDK standards `new ArrayList<>` or `Collections.emptyList()`

Comment (inngvar) [reply generators/server/templates/quarkus/src/main/java/package/service/dto/ManagementInfoDTO.java.ejs:32]: Collections.emptyList() - immutable, but i want modify it in future.
No benefits for now, just a habit. I will change it to new ArrayList<>

**Comment by avdev4j** [comment]
Do you think we can merge this one?

**Comment by joewhite101** [comment]
I think so. Looks like the comments from @danielpetisme were addressed. We can merge and open new tickets if there is more. 

Thanks @inngvar !

**Comment by inngvar** [comment]
claim for bounty https://opencollective.com/generator-jhipster/expenses/25214

**Pull request merged**